tree/dataframe/src/RNTupleDS.cxx

```diff
@@ -160,14 +160,13 @@ void RNTupleDS::SetNSlots(unsigned int nSlots)
       fValuePtrs[i].resize(nColumns, nullptr);
    }
 }
+} // ns Experimental
+} // ns ROOT
 
 
-RDataFrame MakeNTupleDataFrame(std::string_view ntupleName, std::string_view fileName)
+ROOT::RDataFrame ROOT::Experimental::MakeNTupleDataFrame(std::string_view ntupleName, std::string_view fileName)
 {
-   auto pageSource = Detail::RPageSource::Create(ntupleName, fileName);
+   auto pageSource = ROOT::Experimental::Detail::RPageSource::Create(ntupleName, fileName);
    ROOT::RDataFrame rdf(std::make_unique<RNTupleDS>(std::move(pageSource)));
    return rdf;
 }
-
-} // ns Experimental
```

Comment:
The opening namespace should be removed as well, no?

Reply:
no I just moved the closing braces up a bit :D

Reply:
I think class methods can stay in `namespace ROOT { namespace Experimental {`